# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tvtime/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur tvtime-1.0.2-orig/plugins/greedyh.asm tvtime-1.0.2/plugins/greedyh.asm
--- tvtime-1.0.2-orig/plugins/greedyh.asm	2005-08-14 16:16:43.000000000 +0000
+++ tvtime-1.0.2/plugins/greedyh.asm	2006-04-01 21:31:27.554365750 +0000
@@ -18,7 +18,7 @@
 
 #include "x86-64_macros.inc"
 
-void DScalerFilterGreedyH::FUNCT_NAME(TDeinterlaceInfo* pInfo)
+void FUNCT_NAME(TDeinterlaceInfo* pInfo)
 {
     int64_t i;
     bool InfoIsOdd = (pInfo->PictureHistory[0]->Flags & PICTURE_INTERLACED_ODD) ? 1 : 0;
diff -Nur tvtime-1.0.2-orig/plugins/tomsmocomp/TomsMoCompAll2.inc tvtime-1.0.2/plugins/tomsmocomp/TomsMoCompAll2.inc
--- tvtime-1.0.2-orig/plugins/tomsmocomp/TomsMoCompAll2.inc	2004-10-20 15:31:05.000000000 +0000
+++ tvtime-1.0.2/plugins/tomsmocomp/TomsMoCompAll2.inc	2006-04-01 21:45:51.680370250 +0000
@@ -5,9 +5,9 @@
 #endif
 
 #ifdef USE_STRANGE_BOB
-#define SEARCH_EFFORT_FUNC(n) DScalerFilterTomsMoComp::SEFUNC(n##_SB)
+#define SEARCH_EFFORT_FUNC(n) SEFUNC(n##_SB)
 #else
-#define SEARCH_EFFORT_FUNC(n) DScalerFilterTomsMoComp::SEFUNC(n)
+#define SEARCH_EFFORT_FUNC(n) SEFUNC(n)
 #endif
 
 int SEARCH_EFFORT_FUNC(0)		// we don't try at all ;-)
diff -Nur tvtime-1.0.2-orig/plugins/tomsmocomp.cpp tvtime-1.0.2/plugins/tomsmocomp.cpp
--- tvtime-1.0.2-orig/plugins/tomsmocomp.cpp	2004-10-20 17:38:04.000000000 +0000
+++ tvtime-1.0.2/plugins/tomsmocomp.cpp	2006-04-01 21:38:16.927950000 +0000
@@ -31,7 +31,7 @@
 
 #define IS_MMX
 #define SSE_TYPE MMX
-#define FUNCT_NAME DScalerFilterTomsMoComp::filterDScaler_MMX
+#define FUNCT_NAME filterDScaler_MMX
 #include "tomsmocomp/TomsMoCompAll.inc"
 #undef  IS_MMX
 #undef  SSE_TYPE
@@ -39,7 +39,7 @@
 
 #define IS_3DNOW
 #define SSE_TYPE 3DNOW
-#define FUNCT_NAME DScalerFilterTomsMoComp::filterDScaler_3DNOW
+#define FUNCT_NAME filterDScaler_3DNOW
 #include "tomsmocomp/TomsMoCompAll.inc"
 #undef  IS_3DNOW
 #undef  SSE_TYPE
@@ -47,7 +47,7 @@
 
 #define IS_SSE
 #define SSE_TYPE SSE
-#define FUNCT_NAME DScalerFilterTomsMoComp::filterDScaler_SSE
+#define FUNCT_NAME filterDScaler_SSE
 #include "tomsmocomp/TomsMoCompAll.inc"
 #undef  IS_SSE
 #undef  SSE_TYPE
